client/my-sites/site-settings/form-discussion.jsx

```diff
@@ -70,332 +78,292 @@ module.exports = React.createClass( {
-									disabled={ this.state.fetchingSettings }
-									onClick={ this.recordEvent.bind( this, 'Clicked Automatically Close Days Field' ) }
-									onKeyPress={ this.recordEventOnce.bind( this, 'typedAutoCloseDays', 'Typed in Automatically Close Days Field' ) }
-									/>
+									numberOfDays: <FormTextInput
+										name="close_comments_days_old"
+										type="number"
+										min="0"
+										step="1"
+										id="close_comments_days_old"
+										className="small-text"
+										valueLink={ this.linkState( 'close_comments_days_old' ) }
+										disabled={ this.state.fetchingSettings }
+										onClick={ this.recordEvent.bind( this, 'Clicked Automatically Close Days Field' ) }
+										onKeyPress={ this.recordEventOnce.bind( this, 'typedAutoCloseDays', 'Typed in Automatically Close Days Field' ) } />
 								}
 							} )
 						}</span>
-				</label>
-				<label>
-					<input
-					name="thread_comments"
-					type="checkbox"
-					checkedLink={ this.linkState( 'thread_comments' ) }
-					disabled={ this.state.fetchingSettings }
-					onClick={ this.recordEvent.bind( this, 'Clicked Enable Threaded Checkbox' ) }
-					/>
+				</FormLabel>
+				<FormLabel>
+					<FormCheckbox
+						name="thread_comments"
+						checkedLink={ this.linkState( 'thread_comments' ) }
+						disabled={ this.state.fetchingSettings }
+						onClick={ this.recordEvent.bind( this, 'Clicked Enable Threaded Checkbox' ) } />
 					<span>{
 						this.translate( 'Enable threaded (nested) comments up to {{number /}} levels deep', {
 							components: {
-								number: <select
-								className="is-compact"
-								name="thread_comments_depth"
-								valueLink={ this.linkState( 'thread_comments_depth' ) }
-								disabled={ this.state.fetchingSettings }
-								onClick={ this.recordClickEventAndStop.bind( this, 'Selected Comment Nesting Level' ) }
-								>
-									<option value="2">2</option>
-									<option value="3">3</option>
-									<option value="4">4</option>
-									<option value="5">5</option>
-									<option value="6">6</option>
-									<option value="7">7</option>
-									<option value="8">8</option>
-									<option value="9">9</option>
-									<option value="10">10</option>
-								</select>
+								number: <FormSelect
+									className="is-compact"
+									name="thread_comments_depth"
+									valueLink={ this.linkState( 'thread_comments_depth' ) }
+									disabled={ this.state.fetchingSettings }
+									onClick={ this.recordClickEventAndStop.bind( this, 'Selected Comment Nesting Level' ) }>
+										<option value="2">2</option>
+										<option value="3">3</option>
+										<option value="4">4</option>
+										<option value="5">5</option>
+										<option value="6">6</option>
+										<option value="7">7</option>
+										<option value="8">8</option>
+										<option value="9">9</option>
+										<option value="10">10</option>
```

Comment:
This may not be in the scope of this PR, but this might be a good spot to use a map…

```js
import range from 'lodash/utility/range';

<FormSelect …>
	{ range( 2, 10 ).map( level =>
		<option value={ level }>{ level }</option>
	) }
</FormSelect>

// or without lodash
[ 2, 3, 4, 5, 6, 7, 8, 9, 10 ].map( level => … );
```